### DISCUSSION TITLE
Internal Server Error page

### DISCUSSION THREAD

**Discussion by ziyaaktas** [vercel/next.js#14730] [Help] [answered]
When I am running nextjs if a page has a missing import or something, I get Internal Server Error page.  Is there a way to customize this?  I tried _error.js , but no luck.

**Comment by jamesmosier** [comment, marked as answer]
That would typically be a build error that users would never see. Therefore as far as I know there is no customization for this as it is development mode only.

**Reply by ziyaaktas** [reply]
sounds good.  thanks.
